libs/utils/android/workloads/youtube.py

```diff
@@ -77,8 +77,8 @@ def run(self, exp_dir, video_url, video_duration_s, collect=''):
 
         # Stop energy collection
         if 'energy' in collect and self.te.emeter:
-            nrg_data, nrg_file = self.te.emeter.report(exp_dir)
-            logging.info("Estimated energy: %7.3f", float(nrg_data['BAT']))
+            er = self.te.emeter.report(exp_dir)
+            logging.info("Estimated energy: %7.3f", float(er.channels['BAT']))
```

Comment:
well... that's somehow correlated with this channels renaming map.
I think we should find a way to remove this hardcoded "BAT" from workloads which was used to run on a specific AEP configuration ;-)

Maybe we can expose a "channels_map" attribute in the run method and here, for each of the channels listed by this parameter, report the energy consumption?